src/octokit.ts

```diff
@@ -12,8 +12,9 @@ limitations under the License.
 */
 
 import * as core from '@actions/core';
+import * as fetch from '@adobe/node-fetch-retry';
 import { Octokit } from '@octokit/rest';
 import { getOctokit } from '@actions/github';
 
 const githubToken = core.getInput('github_token', { required: true });
-export const octokit = getOctokit(githubToken, { userAgent: 'github-helpers' }).rest as unknown as Octokit;
```

Comment:
i'm guessing we don't need `userAgent` anymore?

Reply:
I think octokit adds a default user agent anyways